yorkie/src/main/kotlin/dev/yorkie/api/ChangeConverter.kt

```diff
@@ -70,13 +73,16 @@ internal fun PBChangePack.toChangePack(): ChangePack {
 }
 
 internal fun ChangePack.toPBChangePack(): PBChangePack {
-    return PBChangePack.newBuilder().apply {
-        documentKey = this@toPBChangePack.documentKey
-        checkpoint = this@toPBChangePack.checkPoint.toPBCheckPoint()
-        snapshot = this@toPBChangePack.snapshot
-        minSyncedTicket = this@toPBChangePack.minSyncedTicket?.toPBTimeTicket()
-        this@toPBChangePack.changes.toPBChanges().forEachIndexed { index, change ->
-            setChanges(index, change)
+    val changePack = this
+    return changePack {
+        documentKey = changePack.documentKey
+        checkpoint = changePack.checkPoint.toPBCheckPoint()
+        snapshot = changePack.snapshot ?: ByteString.EMPTY
```

Comment:
pass empty string if `ChangePack.snapshot` is null
(related discussion: https://github.com/yorkie-team/yorkie-ios-sdk/pull/24#discussion_r1004255928)